Documentation/Signoffs/Resistance_System_Control.md

```diff
@@ -0,0 +1,218 @@
+     Nominal linear actuator voltage = 12 V
+     AMASS XT30 connector current rating = 15 A
+     Imax_continuous(Q1,Q2) = 24 A
+     Imax_continuous(Q3,Q4) = 45 A
+  
+Then:
+
+     Stall (worst-case) resistance = 1.714 Ω
+
+By simulation:
+
+     Pd(Q1,Q2) = 1.05 W  < 78 W  = Pmax(Q1,Q2)
+     Pd(Q3,Q4) = 0.601 W < 2.5 W = Pmax(Q3,Q4)
+
+Constraint met.
+
+PWM Control Simulation Details:
+
+    Duty Cycle = 25 %
+    Switching Frequency = 24 kHz
+    Motor Stalled
+    Polarity (directional) change at 300 ms
+    Switching delay time: 208 to 300 ms
+    
+![spicesim](https://user-images.githubusercontent.com/118228609/216801981-c4e95213-1aef-4920-8c6b-d191272dd4c3.png)
+Figure 3: the LTSPICE schematic prepared for this SPICE simulation.
+
+![loadcurrent](https://user-images.githubusercontent.com/118228609/216802353-ba9df2ad-8ac5-40e6-991e-00e754869501.png)
+Figure 4: the load current. The polarity reverses as expected during the simulation of the directional change starting at 300 microseconds. During the high portions of the PWM cycle, the load current is close to the ideal value of 7 A, meaning the H-bridge produces minimal voltage drop.
+
+Note: It is impossible to fully simulate a motor in SPICE without knowing several mechanical characteristics (moment of inertia, coefficient of viscous friction, and torque) [2] and this information is not given by the actuator manufacturer, so a worst-case non-reactive approximation was used. Back-emf simulation impossible for this reason.
+
+#### Wireless Communications
+
+Given:
+
+    Ts < 100 ms = 0.1 s
+    Distance resolution: 10 bits (limited by microcontroller ADC)
+    Communication protocol: serial
+    
+Then:
+
+    Communication frequency > 1/(0.1 s) = 10 Hz
+    Required samples to update distance variable = (10 bits)/(1 bit/sample) = 10 samples
+    Required baud rate: (10 Hz)*(10 samples) = 100 Baud < 9600 = HC-06 default baud rate
+
+#### Back-EMF Protection
+
+    Vf(D1,D2,D3,D4) = 0.5 V (approximately)
+    Vf(Q1,Q2,Q3,Q4) = 1 V (approximately)
+    Rated Peak Current(D1,D2,D3,D4) = 100 A
+
+Specified in datasheet for free-wheeling applications such as this; constraint met.
+
+#### Mountability
+
+Four 2.5 mm diameter screw holes have been included in the PCB design (see fig. 2). These screw holes can accommodate #3 screws. Mounting and housing cutouts to be detailed in motor stand signoff.
+
+#### Stability
+
+Arduino programmable as PID controller [3].
+
+Manufacturer-provided PID constants [4]:
+
+    Kp = 1.80
+    Ki = 0.0900
+    Kd = 2.50
+
+The above can reasonably be assumed to produce a stable system; therefore, constraint met.
+
+#### Resolution
+
+Given [Resistance System Signoff]:
+
+    Full actuator range = 50.04 mm
+    Tmax(min speed) = 49.23 N•m
+    Tmin(max speed) = 0.004 N•m
+
+And:
+
+    Precision of controller: 10 bits
+
+Then:
+
+    Unique states across full range = 2^10 = 1024
+    Distance precision = (50.04 mm)/1024 = 48.87 μm
```

Comment:
This is not realistic. Further, you need to show that 85 states can be achieved at the slowest pedaling speed and then show that 85 states can be achieved at the max pedaling speed. 

I do not care that the actuator moves. That is not the constraint. The constraint is there are 85 resistance states at each speed which lie between the min and max torque.

It is not at all correct that all states positions of the actuator will map to valid states for all speeds of the tire.  

Avoid formalisms unless they are helpful. Your lemma is not complete and it is unused.

Reply:
> This is not realistic. Further, you need to show that 85 states can be achieved at the slowest pedaling speed and then show that 85 states can be achieved at the max pedaling speed.
>
> It is not at all correct that all states positions of the actuator will map to valid states for all speeds of the tire.
That is not what I stated. Since I cannot explain in detail in the analysis, allow me to explain here. The purpose of the MATLAB program I mentioned was to find the smallest physical distance producing the resistance range deemed usable across the entire range of speeds. This program evaluated both the maximum and minimum speeds in the course of execution. **The smallest distance was determined to be 7.615 mm**. This minimum distance occurred at the slowest speed, meaning the usable distance for all other speeds was greater. Based on the fraction of the actuator's range occupied by the above distance, it was possible to determine the determine **the number of states within this physical distance: 155.**

Reply:
> I do not care that the actuator moves. That is not the constraint. The constraint is there are 85 resistance states at each speed which lie between the min and max torque.

Information auxiliary to this conclusion was included to show how it might be reached. I will include the formula on which the MATLAB script was based to further illustrate how these variables are relevant.